mxcube3/routes/SampleCentring.py

```diff
@@ -601,8 +601,29 @@ def accept_centring():
 
 @mxcube.route("/mxcube/api/v0.1/sampleview/centring/reject", methods=['PUT'])
 def reject_centring():
-    """
-    Reject the centring position.
-    """
+    """Reject the centring position."""
     mxcube.diffractometer.rejectCentring()
     return Response(status=200)
+
+
+@mxcube.route("/mxcube/api/v0.1/sampleview/moveto", methods=['PUT'])
+def pos_from_x_y():
```

Comment:
kind of... but it only tries to move to the center of the image, it ignores anything related to the beam position, just to the center.
Regarding phiy and phiz roles, well, that's my concern, I see that the CentringMath defines those, so we can pick from there, but you are not using it yet.

What would be needed in order to accept this PR? Make clear that this is not _movetobeam_ but _movetocenter?_

I will have a look at those methods.